portfolio/src/main/webapp/script.js

```diff
@@ -45,3 +46,18 @@ function createAnyElement(tag, text) {
   textElement.innerText = text;
   return textElement;
 }
+
+/**
+ * Deletes all comments from the database.
+ */
+async function deleteComments() {
+  const commentsContainer = document.getElementById('comments-container').value;
+  const response = await fetch('/list-comments?max-comments=' + maxComments);
+  const comments = await response.json();
```

Comment:
Curious, do we need to `await` the response.json()? Or can we access the value directly without the `await`?

Reply:
This part is just sloppy work on my part - in fact, this doesn't look like it would compile because maxComments is undefined. It looks like I meant to base the structure of deleteComments() off of getComments(), but stopped halfway through editing. I've since corrected this function in the branch after this one, so I'll bring it up to date.